core/domain/skill_jobs_one_off_test.py

```diff
@@ -169,3 +169,43 @@ def test_migration_job_converts_old_skill(self):
         expected = [[u'skill_migrated',
                      [u'1 skills successfully migrated.']]]
         self.assertEqual(expected, [ast.literal_eval(x) for x in output])
+
+    def test_migration_job_skips_with_new_structure_editors_disabled(self):
```

Comment:
Should this be: `test_migration_job_skips_when_new_structure_editor_is_disabled`?

Reply:
Yes done